configs/jsactions/rollup.config.js

```diff
@@ -130,8 +130,6 @@ export default async args => {
 const nativeExternal = [
     /^mendix\//,
     /^react-native(\/|$)/,
```

Comment:
Ah, so it was removed again. These two commits should be rebased out of the branch